src/pages/trading-specification/components/_popup_menu.tsx

```diff
@@ -48,11 +53,16 @@ const PopUpMenu = ({ market, toggle }: TPopUpMenuProps) => {
     useEffect(() => {
         market_specification.map((specification) => {
             if (specification.market === market) {
-                setMarketsData(specification.dl_data)
-                setDlTitle(specification.dl_title)
+                if (popupType === 'dl') {
```

Comment:
possible declare 'dl' and 'swf' in a const variable and use ?
const SWF:string ='swf'